src/services/rdsDbInstance/connections.ts

```diff
@@ -73,8 +91,173 @@ export default ({
+   */
+  const route53HostedZones = data.find(
+    ({ name }) => name === services.route53HostedZone
+  )
+  if (route53HostedZones?.data?.[globalRegionName]) {
+    const route53HostedZonesInRegion = route53HostedZones.data[
+      globalRegionName
+    ].filter(
+      ({ Id }: RawAwsRoute53HostedZone) =>
+        Endpoint?.HostedZoneId && Id.includes(Endpoint.HostedZoneId)
+    )
+    if (!isEmpty(route53HostedZonesInRegion)) {
+      for (const route53HostedZone of route53HostedZonesInRegion) {
+        connections.push({
+          id: getHostedZoneId(route53HostedZone.Id),
+          resourceType: services.route53HostedZone,
+          relation: 'child',
+          field: 'route53HostedZone',
+        })
+      }
+    }
+  }
+
+  /**
+   * Find KMS
+   * related to this RDS Cluster
+   */
+  const kms: {
+    name: string
+    data: { [property: string]: AwsKms[] }
+  } = data.find(({ name }) => name === services.kms)
+
+  if (kms?.data?.[region]) {
+    // set storage encryption kms key
+    let kmsInRegion: AwsKms[] = kms.data[region].filter(
+      ({ Arn }: AwsKms) => Arn === KmsKeyId
+    )
+    if (!isEmpty(kmsInRegion)) {
+      for (const instance of kmsInRegion) {
+        connections.push({
+          id: instance.KeyId,
+          resourceType: services.kms,
+          relation: 'child',
+          field: 'storageEncryptedKms',
+        })
+      }
+    }
+
+    // set activity stream kms key
+    kmsInRegion = kms.data[region].filter(
+      ({ Arn }: AwsKms) => Arn === ActivityStreamKmsKeyId
+    )
+    if (!isEmpty(kmsInRegion)) {
+      for (const instance of kmsInRegion) {
+        connections.push({
+          id: instance.KeyId,
+          resourceType: services.kms,
+          relation: 'child',
+          field: 'activityStreamKms',
+        })
+      }
+    }
+
+    // set performance insights kms key
+    kmsInRegion = kms.data[region].filter(
+      ({ Arn }: AwsKms) => Arn === PerformanceInsightsKMSKeyId
+    )
+    if (!isEmpty(kmsInRegion)) {
+      for (const instance of kmsInRegion) {
+        connections.push({
+          id: instance.KeyId,
+          resourceType: services.kms,
+          relation: 'child',
+          field: 'performanceInsightsKms',
+        })
+      }
+    }
+  }
+
+  /**
+   * Find IAM Role
+   * related to this RDS Cluster
+   */
+  const iamRoles: {
+    name: string
+    data: { [property: string]: RawAwsIamRole[] }
+  } = data.find(({ name }) => name === services.iamRole)
+
+  if (iamRoles?.data?.[globalRegionName]) {
+    // set associated iam roles
+    let iamRolesInRegion: RawAwsIamRole[] = iamRoles.data[
+      globalRegionName
+    ].filter(({ Arn }: RawAwsIamRole) =>
+      associatedRoles.find(r => r.RoleArn === Arn)
+    )
+    if (!isEmpty(iamRolesInRegion)) {
+      for (const instance of iamRolesInRegion) {
+        connections.push({
+          id: instance.Arn,
+          resourceType: services.iamRole,
+          relation: 'child',
+          field: 'associatedRoles',
```

Comment:
I will need the string `iamRole` as part of the field to handle requirements for EP